ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ErrorFactoriesSpec.scala

```diff
@@ -562,6 +689,27 @@ class ErrorFactoriesSpec
     assertV2Error(error(errorFactoriesV2))(v2_code, v2_message, v2_details)
   }
 
+  private def assertVersionedError(
+      error: ErrorFactories => Status
+  )(
+      v1_code: Code,
+      v1_message: String,
+      v1_details: Seq[Any],
+      v2_code: Code,
+      v2_message: String,
+      v2_details: Seq[ErrorDetails.ErrorDetail],
+  )(implicit dummyImplicit: DummyImplicit): Unit = {
```

Comment:
`DummyImplicit`?

Reply:
So that scalac can discern between this and the above method as they differ in generic type parameter (Function1) only

Reply:
Please use different names then